### PR TITLE
Release 1.30.0

### PR DESCRIPTION
**Pull request by ywkim312**
- [ ] incore-doc: 4.12.0
- [ ] api-doc: 4.12.0
- [ ] pyincore: 1.19.0
- [ ] pyincore-viz: 1.10.1
- [ ] pyincore-data: 0.6.3
- [ ] datawolf: 1.15.0 
- [ ] incore-lab: 1.7..0
- [ ] incore-service: ?
- [ ] semantic-service: ?
- [ ] incore-ui: 1.12.0

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -539,38 +544,48 @@ postgresql:
...
+        http:
+          paths:
+            - path: /auth/
+              pathType: ImplementationSpecific
+  service:
+    type: ClusterIP
+  postgresql:
+    enabled: false
+  externalDatabase:
+    host: incore-postgresql
```
Comment (robkooper) [inline values.yaml:568]: can you add a comment here that if the chart name is changed this needs to change as well.

File: values.yaml [hunk 2]
```diff
@@ -102,6 +102,11 @@ quota: |
      }
   }
 
+ga_key: |
+  {
+    "GA_KEY": "GA_KEY"
```
[the remark this answers is not in the document]
Comment (ywkim312) [reply values.yaml:107]: Replaced and test in dev and it worked well.

File: values.yaml [hunk 3]
```diff
@@ -539,38 +544,48 @@ postgresql:
...
+          paths:
+            - path: /auth/
+              pathType: ImplementationSpecific
+  service:
+    type: ClusterIP
+  postgresql:
+    enabled: false
+  externalDatabase:
+    host: incore-postgresql
+    user: keycloak
```
[the remark this answers is not in the document]
Comment (ywkim312) [reply values.yaml]: added comment

File: values.yaml (hunk 1 above)
Comment (ywkim312) [reply values.yaml]: added comment

File: values.yaml [hunk 4]
```diff
@@ -539,38 +547,48 @@ postgresql:
 # KEYCLOAK
 # ----------------------------------------------------------------------
 keycloak:
-  user: keycloak
-  password: keycloak
-  postgresql:
-    enabled: false
+  httpRelativePath: /auth/
+  auth:
+    adminUser: admin
```
Comment (robkooper) [inline values.yaml:552]: not today, but we should start to use external secrets for all of this.

**Review by robkooper — APPROVED** [review]

**Pull request merged**